src/js/actions/AppsActions.js

```diff
@@ -135,11 +135,12 @@ var AppsActions = {
   },
   applySettingsOnApp: function (appId, settings, isEditing = false) {
     // Version key is not allowed and not needed on settings object
-    delete settings.version;
+    var clonedSettings = Object.assign({}, settings);
```

Comment:
Yeah, but for cloning the notation is a little bit confusing:
```
Util.extendObject(settings);
```
This works, but which object I am going to extend here? :)